src/Transports/MassTransit.EventHubIntegration/EventHubIntegration/ProcessorLockContext.cs

```diff
@@ -52,14 +51,28 @@ public Task Complete(ProcessEventArgs eventArgs)
             return Task.CompletedTask;
         }
 
-        public async Task OnPartitionInitializing(PartitionInitializingEventArgs eventArgs)
+        public async ValueTask DisposeAsync()
+        {
+        }
```

Comment:
```suggestion
        public ValueTask DisposeAsync()
        {
            return ValueTask.CompletedTask;
        }
```
To avoid the async state-machinery.